databases.md

```diff
@@ -0,0 +1,31 @@
+# DATABASES
+
+
+## Prerequisite knowledge:
+NONE
+
+
+## Classification / Acceptance :
+
+#### Apprentice aka _Hello World_
+ * I know what a database is and what it is used for
+ * I can enumerate at least 5 advantages and disadvantages of using a database
+ * I know general requirements that each database must meet to work correctly
```

Comment:
I'm not sure about this point.
what do you mean by database? Is it database engine?
What kind of requirements? hardware requirements? 
By each you mean every class of databases (sql, nosql, graph etc.)? Or database engine (MySQL, PostgreSQL, mongo...)

Reply:
Thank you for that remark. I read it once again and it sounds for me now that it indeed does not bring any information and I am not quite sure what exactly did I mean when I was writing it. I will remove that point.